packages/dds/tree/package.json

```diff
@@ -77,6 +77,7 @@
 		"@fluidframework/runtime-definitions": ">=2.0.0-internal.4.1.0 <2.0.0-internal.4.2.0",
 		"@fluidframework/runtime-utils": ">=2.0.0-internal.4.1.0 <2.0.0-internal.4.2.0",
 		"@fluidframework/shared-object-base": ">=2.0.0-internal.4.1.0 <2.0.0-internal.4.2.0",
+		"@fluidframework/telemetry-utils": ">=2.0.0-internal.4.1.0 <2.0.0-internal.4.2.0",
```

Comment:
why did this move?

Reply:
We need `TelemetryNullLogger` as part of "shipped code" technically now (although we don't publish and the compressor isn't used anywhere in the tree directory. This can go back to devDeps as soon as the compressor gets merged into the runtime.

Reply:
Oh - I didn't even realize it was in a different section, I thought it just got moved upwards in the same list. I'm not a fan of github's "collapse the code in between things during review because it's definitely not relevant" policy